images/c4_container.puml

```diff
@@ -0,0 +1,33 @@
+@startuml C4-Container
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+Container(seed, "Main Seedcase", "web-interface")
+Container(sec, "User Authentication", "interface", "called when selecting any app which needs security")
+Container(sprout, "Sprout", "web-app", "uploads the research instance data")
+Container(flower, "Flower", "web-app", "makes metadata available to browse")
+Container(prop, "Propagate", "web-app", "allows others to request data")
+Container(garden, "Garden", "web-app", "handles workflow for data requests")
+Container_Boundary(backend, "data storage"){
+  ContainerDb(data, "Research Data", "db, csv, etc")
+  ContainerDb(user, "User Data", "db")
+}
```

Comment:
```suggestion
Container_Boundary(seed, "Seedcase Data Resource", "web-interface") {
	Container(sec, "User Authentication", "interface", "called when selecting any app which needs security")
	Container(sprout, "Sprout", "web-app", "uploads the research instance data")
	Container(flower, "Flower", "web-app", "makes metadata available to browse")
	Container(prop, "Propagate", "web-app", "allows others to request data")
	Container(garden, "Garden", "web-app", "handles workflow for data requests")
	Container_Boundary(backend, "data storage"){
  	ContainerDb(data, "Research Data", "db, csv, etc")
  	ContainerDb(user, "User Data", "db")
	}
}
```
Change "Main Seedcase" to a Container boundary and call it "Seedcase Data Resource" to fit with the Context diagram? 
And then change the relationships accordingly, so User authentication goes to Sprout, for instance.